sources/packages/backend/apps/load-test-gateway/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Base Image
+FROM artifacts.developer.gov.bc.ca/redhat-access-docker-remote/ubi8/nodejs-18:1-71.1697652955
+
+LABEL maintainer="BC GOV"
+
+# Copying root files to image workdir.
+COPY /*.json /*.js ./
+
+# Installing dependencies
+RUN npm ci
+
+# Copying sources.
+COPY ./apps/load-test-gateway ./apps/load-test-gateway
+COPY ./libs ./libs
+
+# Building app
+RUN npm run build load-test-gateway
+
```

Comment:
Are we missing `EXPOSE ${PORT}` here or is there a reason to not have it added?

Reply:
Correct me if wrong, I am not sure about the impact of adding expose, as queue-consumers(bull dashboard) is working without the expose and I was assuming the same for here as well. may be I am missing something

Reply:
As we checked, the `yaml` has the port expose 😉